src/reducers/filters.js

```diff
@@ -0,0 +1,16 @@
+import { CHANGE_FILTER } from '../constants';
+
+const initialFilterState = '';
```

Comment:
- this should default to 'ALL' category

Reply:
![image](https://user-images.githubusercontent.com/52007960/91348427-5a60bd00-e7a9-11ea-970a-aaf6b9a3942d.png)

Reply:
Hello! Thank you for your observations. The pathwright actually tells us to keep the ALL category **outside**  the react store!